app/uploaders/report_photo_uploader.rb

```diff
@@ -0,0 +1,13 @@
+class ReportPhotoUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :fog
+
+  version :thumb do
+    process resize_to_fill: [200, 200]
```

Comment:
These are generated on demand or created and added to bucket in advance?

Reply:
With carrierwave thumbs are created in advance. Another option would be to use Refile or Dragonfly (https://infinum.co/the-capsized-eight/best-rails-image-uploader-paperclip-carrierwave-refile). They support file resizing on demand. Haven't tried them though.

Reply:
Ok. Let's keep it for now.

Reply:
It would be good to define thumb size. I just added 200x200, but we could set a real value. or have multiple thumbs, if needed.